src/components/PostListRow/PostListRow.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import cx from 'classnames'
+import { isEmpty } from 'lodash/fp'
+import { personUrl, topicUrl } from 'util/navigation'
+import { humanDate } from 'hylo-utils/text'
+import Avatar from 'components/Avatar'
+import Icon from 'components/Icon'
+import './PostListRow.scss'
+
+const PostListRow = (props) => {
+  const {
+    routeParams,
+    post,
+    showDetails,
+    voteOnPost,
+    expanded
+  } = props
+  const {
+    title,
+    details,
+    creator,
+    createdAt,
+    commentersTotal,
+    votesTotal,
+    myVote,
+    topics
+  } = post
+
+  if (!creator) { // PostCard guards against this, so it must be important? ;P
+    return null
+  }
+
+  const creatorUrl = personUrl(creator.id, routeParams.slug)
+  const numOtherCommentors = commentersTotal - 1
+  const unread = false
+
+  return (
+    <div styleName={cx('post-row', { unread, expanded })} onClick={showDetails}>
+      <div styleName='votes'>
+        <a onClick={voteOnPost} styleName={cx('vote-button', { voted: myVote })}
+          data-tip-disable={myVote} data-tip='Upvote this post so more people see it.' data-for='postfooter-tt'>
```

Comment:
not always seeing this tooltip for some reason

Reply:
Fixed, I was trying to reuse an existing ID but it's not always rendered!